### ISSUE TITLE
no android version

### ISSUE THREAD

**Issue by bitekirosan** [TheBetaM/CrateModLoader#3]
i know this may sound annoying, but would it be possible to make an android (APK) version of the crate mod loader?

**Comment by TheBetaM** [comment]
Yes, but with probably less games/consoles supported (features and Mod Menu stuff should be the same). This is something I'd like to expand on eventually.

**Comment by bitekirosan** [comment]
i'd be ok with less features/consoles supported as long as i could still
mod the games without needing a computer, since i don't have one.

Em seg, 8 de fev de 2021 11:13, BetaM <notifications@github.com> escreveu:

**Comment by bitekirosan** [comment]
> Yes, but with probably less games/consoles supported (features and Mod Menu stuff should be the same). This is something I'd like to expand on eventually.

i'd be ok with less features/consoles supported as long as i could still
mod the games without needing a computer, since i don't have one.
